open-notes/maturity-model.md

```diff
@@ -0,0 +1,394 @@
+- [ ] [Uptime alerts](#uptime-alerts) [observability]
+- [ ] [Secrets in key vault](#secrets-in-key-vault) [security]
+- [ ] [Custom metrics and KPIs](#custom-metrics-and-kpis) [observability]
+- [ ] [Limited WIP](#limited-wip) [process]
+- [ ] [Use-case entry points](#use-case-entry-points) [code]
+
+### Level 5
+
+- [ ] [Dashboards III](#dashboards-iii) [observability]
+- [ ] [Acceptance tests](#acceptance-tests) [maintenance]
+- [ ] [Design pitches](#design-pitches) [process, knowledge, communication]
+- [ ] [Tech debt days](#tech-debt-days) [process]
+- [ ] [Vulnerability scans](#vulnerability-scans) [security]
+- [ ] [Idempotent APIs](idempotent-api) [code]
+  
+### Level 6
+
+- [ ] [Keep current](#keep-current) [security, maintenance]
+- [ ] [Command Query Separation](#command-query-separation) [code]
+- [ ] [Dependency aware health checks](#dependency-aware-health-checks) [observability]
+- [ ] [Retro all the things](#retro-all-the-things) [process]
+- [ ] [Deploy daily](#deploy-daily) [process]
+
+### Level 7
+
+- [ ] [Honest types](#honest-types) [code, knowledge]
+- [ ] [Pair programming](#pair-programming) [process, knowledge]
+- [ ] [Correctness alerts](#correctness-alerts) [observability]
+- [ ] [APM tracing](#apm-tracing) [observability]
+- [ ] [Resilient requests](#resilient-requests) [code]
+
+### Level 8
+
+- [ ] [Embrace immutable code](#embrace-immutable-code) [code]
+- [ ] [Mob programming](#mob-programming) [process, knowledge, communication]
+- [ ] [Learning days](#learning-days) [process, knowledge]
+- [ ] [Synthetics](#sythetics) [observability]
+
+### Level 9
+
+- [ ] [Cyclomatic complexity ceiling](#cyclomatic-complexity-ceiling) [code, observability]
+- [ ] [Quantitative team health](#quantitative-team-health) [observability, process]
+- [ ] [Embrace FP principles](#embrace-fp-principles) [code]
+- [ ] [PR to production](#pr-to-production) [process]
+
+### Level 10
+
+- [ ] [Dashboards IV](#dashboards-iv) [observability, process]
+- [ ] [Load testing](#load-testing) [maintenance]
+- [ ] [Immutable Architecture](#immutable-architecture) [code]
+
+### Level 11
+
+- [ ] [Domain Driven design](#domain-driven-design) [process, knowledge, code, communication]
+- [ ] [Property based testing](#property-based-testing) [maintenance]
+- [ ] [Chaos engineering](#chaos-engineering) [code, maintenance]
+
+
+## Skills
+
+### Unit testing
+
+Unit testing are written for all new features. The unit tests are readable, and [easy to maintain](https://devonburriss.me/maintainable-unit-tests/) using an ever growing set of helpers designed specifically for that domain. Unit tests are submitted with the new code or changes they are covered, thus actually ensuring some quality before code is deployed to production.
+
+### Centralized and prioritized stories
+
+The team keeps a single shared list of work that is being worked on and the list is prioritized from highest priority to lowest... preferably based on value, not urgency.
```

Comment:
You touch the point. Value. It is important to have guidelines about what can be defined as value. Is value market share? Profit? NPS? eNPS? Other?